webapp/src/js/directives/content-row.js

```diff
@@ -62,7 +62,14 @@ angular.module('inboxDirectives').directive('mmContentRow', function() {
       primaryContact: '=',
 
       // integer: (optional) how much tasks to show as pending for this contact
-      taskCount: '='
+      taskCount: '=',
+
+      // boolean: (optional) whether the contact is muted
+      // html-minifier considers `muted` to be a boolean attribute, so it will strip its value
+      muted: '=isMuted',
+
+      // boolean: (optional) whether to display the "muted" text along with contact name
+      displayMuted: '='
```

Comment:
I can't see anywhere where we're supplying `isMuted` but setting `displayMuted` to false. Do we need both?

Reply:
Yes. The difference is if we display the `"muted"` test underneath the contact's name. In the LHS it shouldn't be displayed, but it should be displayed in contact cards in Contact profiles (which also use this template).